featherpad/singleton.cpp

```diff
@@ -273,6 +273,15 @@ QStringList FPsingleton::processInfo (const QString& message,
         if (!path.isEmpty()) // no empty path/name
         {
             QString realPath = path;
+#ifdef __OS2__
+						// We try to match, whether the filepath begins with a driverletter + ':'
+						// QUrl mistakes that for a scheme.
+						QRegularExpression re("^[A-Za-z]:");
+						if (path.indexOf(re, 0, nullptr) != -1)
+						{
+							realPath.prepend("file:");
+						}
+#endif
```

Comment:
Please change tabs to soft tabs (`Ctrl+Tab` in FeatherPad).

I hope this works well under OS/2 as the original code works under Linux/Unix.